### PR TITLE
Remove nested objects

### PR DESCRIPTION
**Pull request by LeonMatthesKDAB**
This is a temporary measure meant to make the transition to our new API easier.

Nested objects will change a lot anyway.

### CODE REVIEW

File: book/src/concepts/nested_objects.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # Nested Objects
 
+During the transition to CXX-Qt 0.4, Nested object support is temporarily removed.
+
+We are currently reworking the way nested objects work.
+They will be back, and better than ever 😃.
```
Comment (ahayzen-kdab) [inline book/src/concepts/nested_objects.md:13]: :pray:

**Review by ahayzen-kdab — CHANGES_REQUESTED** [review]
Looks good, can you rebase onto main to remove the first commit.

And look at `examples/README.md` as that needs the "Nesting Rust objects as properties and invokables" line removing ?

**Comment by LeonMatthesKDAB** [comment]
README.md is fixed.

**Review by ahayzen-kdab — APPROVED** [review]
LGTM

**Pull request merged**